src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -20,6 +20,10 @@ trait ThurloeService extends HttpService {
   val dataAccess: DataAccess
   val thurloePrefix = "thurloe"
 
+  // TODO: Relies on swagger existing in resources/swagger/
+  val swaggerSite = path("swagger-site") { getFromResource("swagger/index.html") } ~ getFromResourceDirectory("swagger")
```

Comment:
Whoops, well the swagger-ui exposed endpoint has been removed from cromwell as of a few commits back. We should probably have a test here and in cromwell for checking that `/swagger/index.html` returns the expected content from the ui jar. For now… this line doesn't really do anything but return 404s for various reasons. :wink: [This](https://github.com/broadinstitute/cromwell/commit/f82eaa5cbcc7ed64f534c671c13a2f9cdb1df5f1#diff-e25accda21d47a99a8060d7e9f4d32a5) commit has a ui endpoint that used to serve up the ui jar content. (Note: It also included dynamic spray-swagger generating the swagger json, that I believe we are _definitely_ not using anymore.)